FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -1465,7 +1465,7 @@ def setupSite( self, scriptCfg, cfg = None ):
     setupAgents = [ k.split( '/' ) for k in self.localCfg.getOption( cfgInstallPath( 'Agents' ), [] ) ]
     setupExecutors = [ k.split( '/' ) for k in self.localCfg.getOption( cfgInstallPath( 'Executors' ), [] ) ]
     setupWeb = self.localCfg.getOption( cfgInstallPath( 'WebPortal' ), False )
-    setupWebApp = self.localCfg.getOption( cfgInstallPath( 'WebApp' ), False )
+    setupWebApp = self.localCfg.getOption( cfgInstallPath( 'WebApp' ), True )
```

Comment:
Why ? This installs by default the new portal instead of the old one. I do not think that we advocate the old portal any more. Rather we should drop it altogether.